src/components/Executions/ExecutionRow.js

```diff
@@ -141,7 +143,10 @@ const ExecutionRow = ({
           <Anchor onClick={onToggle}>Toggle results</Anchor>
           {showData && (
             <ResultWrapper className="mc-p-4 mc-my-3">
-              <TestResults testResults={JSON.parse(testResults)} />
+              <TestResults
+                testResults={JSON.parse(testResults)}
+                onRerun={status === statuses.error ? onRerun : null}
```

Comment:
`onRerun` is a function, so it's should receive a `null` object. I'd rather use `undefined` instead of `null`